src/features/compensations/components/CreateCompensations/components/CompensationsTypesTips/CompensationsTypesTips.scss

```diff
@@ -0,0 +1,26 @@
+.compensations-types-tips {
+  margin-top: 16px;
+  width: 100%;
+  font-weight: 400;
```

Comment:
The default font-weight value is 400. It seems that this property can be deleted.